manifests/config.pp

```diff
@@ -131,7 +133,6 @@
     file { 'prometheus.yaml':
       ensure       => present,
       path         => "${prometheus::server::config_dir}/${prometheus::server::configname}",
-      owner        => $prometheus::server::user,
```

Comment:
@anarcat Why did you remove the owner? I am pretty sure this is set on purpose.

Reply:
because i don't want the file to be writable by the prometheus user. services shouldn't be able to modify their own config files.

Reply:
i split this question off into a separate, more elaborate PR in #324